### DISCUSSION TITLE
Highlight Canvas feature

### DISCUSSION THREAD

**Discussion by kobecow** [wojtekmaj/react-pdf#1176] [General]
I looked for a way to draw rectangles on pdf. I found \@react-pdf/render had that ability. Today I found this [discussion](https://github.com/wojtekmaj/react-pdf/issues/979) and understood how to draw rectangle on PDF.

I'd like to add an example of using Canvas in README.md or Recipes.
What do you think ?

**Comment by wojtekmaj** [comment]
What we really need is a way to properly link all the CodeSandboxes, which there are many, to Wiki so the users could click around and see all the examples live, working!
